tpm/tpm.go

```diff
@@ -34,27 +32,9 @@ type Closer interface {
 // extensive resource management that the kernel already handles for us
 // when using the kernel resource manager.
 func Open(path string) (Closer, error) {
-	switch {
-	case isDevNode(path, "tpmrm"):
-		return linuxtpm.Open(path)
-	case isDevNode(path, "tpm"):
-		slog.Warn("direct use of the TPM can lead to resource exhaustion, use a TPM resource manager instead")
-		return linuxtpm.Open(path)
-	default:
-		return nil, fmt.Errorf("unsupported TPM device path: %s", path)
-	}
-}
-
-func isDevNode(path, kind string) bool {
-	if strings.Contains(kind, "/") {
-		return false
-	}
-	prefix := "/dev/" + kind
-
-	if !strings.HasPrefix(path, prefix) {
-		return false
+	if !strings.HasPrefix(path, "/dev/tpmrm") {
```

Comment:
This is currently in a `!windows` file. Do Darwin and BSD also have the same path? (I'm not even going to worry about Solaris, etc.)

Reply:
Removed the build tag entirely upon removing the `Open` func.